cg/models/taxprofiler/taxprofiler.py

```diff
@@ -65,16 +66,17 @@ def headers() -> list[str]:
 
     def reformat_sample_content(self) -> list[list[str]]:
         """Reformat sample sheet content as a list of list, where each list represents a line in the final file."""
-        return [
-            [
+        reformatted_content = []
+        for run_accession, (forward_path, reverse_path) in enumerate(
```

Comment:
I think we can also make use of `start=1` of enumerate, and remove the `+1` from `run_accession`

Reply:
Updated and tested.

Reply:
Nice 👍